workflow.cwl

```diff
@@ -92,20 +92,61 @@ steps:
   #       source: "#submissionId"
   #   out: [config_file]
 
-  download_goldstandard:
-    run: https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/cwl-tool-synapseclient/v1.4/cwl/synapse-get-tool.cwl
+  unzip_generate_config:
+    run: steps/unzip_submission.cwl
     in:
-      # TODO: replace `valueFrom` with the Synapse ID to the challenge goldstandard
-      - id: synapseid
-        valueFrom: "syn58613732"
-      - id: synapse_config
-        source: "#synapseConfig"
+      - id: compressed_file
+        source: "#download_compressed_file/filepath"
```

Comment:
`download_compressed_file` does not currently exist as a step in the workflow.  Typo?  Or maybe a missed commit?